test/spec/token.js

```diff
@@ -1678,15 +1678,148 @@ describe('token.getWithRedirect', function() {
       ],
       expectedRedirectUrl: 'https://auth-js-test.okta.com/oauth2/v1/authorize?' +
                             'client_id=NPSfOkH5eZrTy8PMDlvx&' +
+                            'nonce=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa&' +
                             'redirect_uri=https%3A%2F%2Fexample.com%2Fredirect&' +
-                            'response_type=code&' +
                             'response_mode=form_post&' +
+                            'response_type=code&' +
+                            'sessionToken=testToken&' +
+                            'state=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa&' +
+                            'scope=openid%20email'
+    });
+  });
+
+  it('passes optional login_hint query param through', function() {
```

Comment:
Here and below are the new tests.  

Note there is no update to README because the README already doesn't document the optional params, instead referring to the API docs, so I followed that trend.